src/screens/Login/index.jsx

```diff
@@ -0,0 +1,67 @@
+import { useState } from 'react';
+import { login } from 'services/api/auth';
+
+const Login = () => {
+  const [formErrors, setFormErrors] = useState([]);
+  const [isFormProcessing, setIsFormProcessing] = useState(false);
+
+  const handleSubmit = async (event) => {
```

Comment:
How about moving the `handleSubmit` to a different file? naming may be `handler.js`? What do you think? It will clean up this fille I think.

Reply:
I've extracted some logic out, what do you think? 💭